lagom-client-restaurant/menu-item-impl/src/main/resources/application.prod.conf

```diff
@@ -2,63 +2,18 @@ include "application.conf"
 
 play.http.secret.key = "unused secret"
 
-http {
-  address = ${?HTTP_BIND_ADDRESS}
-  port = 9000
-}
-
-play.filters.hosts {
-  # Requests that are not from one of these hosts will be rejected.
-  allowed = [${?ALLOWED_HOST}]
-}
-akka.discovery.method = kubernetes-api
+# Workaround for https://github.com/lagom/lagom/issues/2239
+akka.discovery.method = akka-dns
```

Comment:
This uses DNS SRV for service-to-service lookups, which is a good default for Kubernetes. In future versions of Lagom, you won't need to set it explicitly.